src/utils/utils_log.c

```diff
@@ -305,18 +305,18 @@ void util_log_init(void) {
         loggerConfig.flushLevel = LOG_FATAL;
     }
 
-#ifdef UMF_SRC_VERSION
+#ifdef UMF_VERSION
 // convert a define to a C string
 #define STR_(X) #X
 #define STR(X) STR_(X)
-#define STR_UMF_SRC_VERSION "src version: " STR(UMF_SRC_VERSION) ", "
-#else /* !UMF_SRC_VERSION */
-#define STR_UMF_SRC_VERSION ""
-#endif /* !UMF_SRC_VERSION */
+#define STR_UMF_VERSION "UMF version: " STR(UMF_VERSION) ", "
+#else /* !UMF_VERSION */
+#define STR_UMF_VERSION ""
+#endif /* !UMF_VERSION */
 
     int umf_ver = umfGetCurrentVersion();
     LOG_INFO(
-        "Logger enabled (umf version: %i.%i, " STR_UMF_SRC_VERSION
+        "Logger enabled (umf version: %i.%i, " STR_UMF_VERSION
```

Comment:
It looks very strange now:
```
util_log_init: Logger enabled (umf version: 0.9, UMF version: 0.1.0-git28.g4ab61a7,
```

util_log_init: Logger enabled (**umf version**: 0.9, **UMF version**: 0.1.0-git28.g4ab61a7,

Reply:
from the 0.9.0 release this will align - I updated RELEASE_STEPS to update this manually for each major and minor release

Reply:
OK, but what about "`umf version`" and the second "`UMF version`"

Reply:
we can rename the first one to "API version" or simply remove the first part (leaving just the full UMF_VERSION) - what do you think...?

Reply:
I would remove the first part, but the second one should be required then:
```c
#else /* !UMF_VERSION */
#error UMF_VERSION not defined!
#endif /* !UMF_VERSION */
```

Reply:
make sense, thx! Done.